packages/sdk/android/src/main/java/com/primerioreactnative/extensions/PrimerOrder.kt

```diff
@@ -1,6 +1,11 @@
 package com.primerioreactnative.extensions
 
 import com.primerioreactnative.datamodels.PrimerOrderRN
+import com.primerioreactnative.datamodels.PrimerShippingRN
 import io.primer.android.domain.action.models.PrimerOrder
+import io.primer.android.domain.action.models.PrimerShipping
 
-internal fun PrimerOrder.toPrimerOrderRN() = PrimerOrderRN(countryCode)
+// TODO enable shipping
```

Comment:
leftover?

Reply:
no, we have to enable this on native first on our side